perception/sandbox/eyehandcal/calibration_points.json

```diff
@@ -1 +1,26 @@
-{"xyz": [[0.7841147184371948, -0.26561641693115234, 0.237627312541008], [0.8307967782020569, 0.02608540467917919, 0.2515111565589905], [0.8506162166595459, -0.02456057071685791, 0.347420334815979], [0.8517714738845825, -0.05609707906842232, 0.4044360816478729], [0.8047099709510803, -0.2907401919364929, 0.3745661973953247], [0.8047105073928833, -0.29073861241340637, 0.37456685304641724]], "quat": [[0.8225067853927612, -0.08389443904161453, 0.4012543261051178, -0.39425790309906006], [0.6798139810562134, -0.3220716714859009, 0.6247647404670715, -0.20926488935947418], [0.5894655585289001, -0.43789276480674744, 0.6651686429977417, -0.13539202511310577]]}
\ No newline at end of file
+{
```

Comment:
Perhaps we should add `indent=4` to [record_calibration_points.py](https://github.com/facebookresearch/fairo/blob/main/perception/sandbox/eyehandcal/src/eyehandcal/scripts/record_calibration_points.py#L82) as well